core/scss/utilities/mixins/gradient/_linear-gradient.scss

```diff
@@ -0,0 +1,30 @@
+@charset 'UTF-8';
+
+//
+// @linear-gradient($direction, $color-stops...)
+//
+// @author Hugo Giraudel
+//
+// Mixin to create a linear gradient with a plain color fallback and -webkit support.
+//
+// **Examples:**
+// - `@include linear-gradient(#31b7d7, #edac7d);`
+// - `@include linear-gradient(to right, #eee 0%, $color-cardinal-red 50%, $color-palo-alto 100%);`
+// - `@include linear-gradient(42deg, #b58234 0%, #d2b545 50%, #d7c04d 50.01%, $color-white 100%);`
+//
+// $direction - string: Linear gradient direction. One of: to top, to top right, to right top, to right, to bottom right, to right bottom, to bottom, to bottom left, to left bottom, to left, to left top, to top left
+// $color-stops - list: Color-stops composing the gradient. Each includes a color value, and a location in percentage (when there are more than 2 $color-stops).
+//
+// Style guide: Mixins.Gradient.linear-gradient
+//
+
+@mixin linear-gradient($direction, $color-stops...) {
+  @if is-direction-or-angle($direction) == false {
+    $color-stops: ($direction, $color-stops);
+    $direction: 180deg;
+  }
+
+  background: nth(nth($color-stops, 1), 1);
+  background: -webkit-linear-gradient(to-legacy-direction($direction), $color-stops);
+  background: linear-gradient($direction, $color-stops);
```

Comment:
I'm thinking (I am assuming you are, too) that we can ignore the related CC issue here, as the prefix is there as a fallback/level of protection. We have similar "errors" on other components, including Alerts and others that call both the `.png` and .svg` image files as `background-image`. One related question: is the prefix still required for modern browsers? Perhaps we want it there anyway to help support older versions of Safari, etc.?

Reply:
@josephgknox That's a great question... I believe the author wrote the mixin a while back. I'm looking at https://caniuse.com/#feat=css-gradients, it looks like only Chrome v4-25, Safari 4-6 and iOS Safari 5-6.1 (the one ones that actually matter) and they all add up to <0.3% global usage. I think I might be ok with removing the -webkit support since gradients are fairly minor enhancements. What do you think? That way I can actually remove some of those helper functions as well.

Reply:
I thought that might be the case. I'm not sure. I don't think we have an established threshold for these things, but I would imagine that <0.3% is below the threshold. I'm good with dropping it. Maybe we get one more vote? @sherakama?

Reply:
Could we get a vote, @sherakama 😺 ? Ok to drop support for legacy -webkit convention? If ok, I'll get rid of the to-legacy-direction function too.

Reply:
Isn't that what our autoprefixer is supposed to take care of? I thought we were avoiding vendor prefixes in our SASS.

Reply:
With linear gradient it's a little trickier. It needs not only a prefix but the direction convention and angle calculation is different (which needs a helper function).

Reply:
Gotcha thanks,

Ok, our compatibility is listed here: https://github.com/SU-SWS/decanter/wiki/Compatibility

I think the browsers you mentioned are old enough that we don't have to support them. In this case, I am good with dropping the extra webkit option.

Reply:
Woot! The vote is in. Soon to be another PR merged.

Reply:
@josephgknox @sherakama -webkit support removed, but I did keep the convert angle helper function since it might come in handy for other things later.